pkgs/games/katawa-shoujo/default.nix

```diff
@@ -0,0 +1,179 @@
+{ stdenvNoCC
+, lib
+, fetchurl
+, autoPatchelfHook
+, freetype
+, makeWrapper
+, libGL
+, libGLU
+# Darwin cannot handle these when devendored:
+# - DYLD_LIBRARY_PATH masks system libraries with similar, differently-cased names and cause missing symbol errors
+# - symlinks cause unrelated BMP image loading to fail(?)
+, devendorImageLibs ? !stdenvNoCC.hostPlatform.isDarwin
+, libjpeg
+, libpng12
+, libX11
+, libXext
+, libXi
+, libXmu
+, runtimeShell
+, SDL
+, withSDL2 ? true
```

Comment:
Why do we need an option for using SDL1?

Reply:
`SDL_compat` gave me problems with some wrapping outside of Nixpkgs, but I've worked around since. Could still be nice to replace the vendored SDL1 libraries with Nixpkgs-built (and possibly more updated) ones but w/e. Non-`SDL_compat` support dropped.